applications/ecallmgr/src/ecallmgr_fs_event_stream_sup.erl

```diff
@@ -16,11 +16,9 @@
         ]).
 -export([init/1]).
 
--define(EVENTS, application:get_env(?APP, 'event_stream', ?FS_EVENTS)).
--define(CUSTOM_EVENTS, application:get_env(?APP, 'event_stream_custom', ?FS_CUSTOM_EVENTS)).
+-define(EVENTS, ?FS_EVENTS).
```

Comment:
Do we need EVENTS now? Is it used elsewhere in this module?

Reply:
finishing this, they should be get from config